src/dom.js

```diff
@@ -112,6 +112,12 @@ export function documentContainsPolyfillInternal_(doc, element) {
  */
 export function removeElement(element) {
   if (element.parentElement) {
+    const activeElement = element.ownerDocument.activeElement;
+    // For accessibility reasons, restore focus to document if the focused
+    // element is inside the removed one.
+    if (element == activeElement || element.contains(activeElement)) {
+      element.ownerDocument.defaultView./*REVIEW*/focus();
```

Comment:
`.focus()` calls need to be wrapped in a try block. At least in IE they can fail.

Probably good to make a `focus(element)` function that does that.